src/main/java/frc/team5115/Classes/Software/PhotonVision.java

```diff
@@ -42,7 +42,17 @@ public class PhotonVision {
      public PhotonVision() {
          // Change the name of your camera here to whatever it is in the PhotonVision UI.
          photonCamera = new PhotonCamera(frc.team5115.Constants.VisionConstants.leftCameraName);
- 
+         photonCamera = new PhotonCamera(frc.team5115.Constants.VisionConstants.rightCameraName);
+         photonCamera = new PhotonCamera(frc.team5115.Constants.VisionConstants.frontCameraName);
```

Comment:
Not sure I understand why you're setting `photonCamera` to three different PhotonCamera objects. The first two are going to be thrown away.